test/extensions/filters/network/dubbo_proxy/utility.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <initializer_list>
+
+#include "common/buffer/buffer_impl.h"
+#include "common/common/byte_order.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+namespace {
```

Comment:
unnamed namespace does almost nothing in header.

Reply:
done